how-to/customize-workspace/client/src/framework/workspace/notifications.ts

```diff
@@ -30,13 +30,15 @@ export async function register(): Promise<RegistrationMetaInfo> {
 			const settings = await getSettings();
 			const notificationPlatformSettings = settings?.notificationProvider;
 			if (notificationPlatformSettings !== undefined) {
-				try {
-					await registerPlatform(notificationPlatformSettings);
-					notificationsRegistered = true;
-					logger.info("Registered notifications");
-				} catch (error) {
-					logger.error("We were unable to register with Notification Center", error);
-				}
+				registerPlatform(notificationPlatformSettings)
```

Comment:
If the intention is for this to be a background promise then a comment saying so would probably be wise, otherwise a well meaning person might just convert it back to the async code. Alternatively keep the async and wrap it in a setTimeout

Reply:
Will add a comment, would rather avoid a setTimeout because it might give the wrong impression.